c++/src/capnp/compiler/capnpc-c++.c++

```diff
@@ -3045,13 +3045,17 @@ private:
             }
           },
           "\n"
+          "CAPNP_BEGIN_HEADER\n"
+          "\n"
           "namespace capnp {\n"
           "namespace schemas {\n"
           "\n",
           KJ_MAP(n, nodeTexts) { return kj::mv(n.capnpSchemaDecls); },
           "\n"
           "}  // namespace schemas\n"
           "}  // namespace capnp\n"
+          "\n"
+          "CAPNP_END_HEADER\n"
```

Comment:
Looks like you've placed this at the end of the schema block, but the main body of the header actually appears after that. The last part of the header is generated at line 3067 below, so you'll want to insert this after that.

Reply:
Should be resolved as of 4df555139a3feaaf9746740cefecf0beadc01ba9